### PR TITLE
world switcher: add ping

### LINKED ISSUE #1597: Request for a ping indicator in the world switcher menu
I believe it would be very useful, and hopefully easy to implement an indicator to show how much ping a world has when hopping. 

Two potential design mock-ups:

image
image

### LINKED ISSUE #2640: Add ping indicator
An infobox like the Timers or Slayer counter that shows ping would be amazing

### PR DESCRIPTION
**Pull request by Adam-**
Fixes #1597
Closes #2640

### CODE REVIEW

**Comment by Adam-** [comment]
@psikoi please review.

File: runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java [hunk 1]
```diff
@@ -715,4 +730,35 @@ private ChatPlayer getChatPlayerFromName(String name)
...
 	}
+
+	@Schedule(
+		period = 10,
+		unit = ChronoUnit.SECONDS,
+		asynchronous = true
+	)
+	public void pingCheck()
+	{
+		if (worldResult == null)
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java]: You should add check if panel is enabled in config here

File: runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java [hunk 2]
```diff
@@ -715,4 +730,35 @@ private ChatPlayer getChatPlayerFromName(String name)
...
+		{
+			lastPingCheck = 0;
+		}
+
+		World world = worlds.get(lastPingCheck++);
+		int ping = Ping.ping(world);
+
+		log.debug("Ping for {}: {}", world.getId(), ping);
+
+		panel.updatePing(world, ping);
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java]: This looks dangerous, move it to swing thread

**Comment by Abextm** [comment]
These bindings look mostly wrong for 32bit runtimes.

**Pull request merged**